### PR TITLE
#0: Add initial t3000 nightly pipeline

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket

https://github.com/tenstorrent-metal/metal-internal-workflows/issues/291

### Problem description

We need a T3000 nightly pipeline and a TG nightly pipeline to support development.

### What's changed

Create pipelines with new paradigms:
- symlinks to tests
- less indirect scripting to call test commands

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by tt-rkim** [comment]
cc: @SeanNijjar 
waiting on review from @ttmchiou

**Review by TT-billteng — APPROVED** [review]

**Review by ttmchiou — APPROVED** [review]

**Comment by SeanNijjar** [comment]
Hey @tt-rkim - is there a corresponding issue for TG nightly?

**Comment by tt-rkim** [comment]
@SeanNijjar : https://github.com/tenstorrent-metal/metal-internal-workflows/issues/291

**Pull request merged**
